eZ/Bundle/EzPublishCoreBundle/Command/NormalizeImagesPathsCommand.php

```diff
@@ -172,14 +191,16 @@ private function updateImagePath(int $fieldId, string $oldPath, string $newPath)
             }
         }
 
-        $this->moveFile($oldFileName, $newFilename, $oldPath);
+        $this->moveFile($oldFileName, $newFilename, $oldBinaryFile, $inputStream);
     }
 
-    private function moveFile(string $oldFileName, string $newFileName, string $oldPath): void
-    {
-        $oldBinaryFile = $this->ioService->loadBinaryFileByUri(\DIRECTORY_SEPARATOR . $oldPath);
+    private function moveFile(
+        string $oldFileName,
+        string $newFileName,
+        BinaryFile $oldBinaryFile,
+        $inputStream
```

Comment:
missing strict type

Reply:
what kind of strict type should I put for `resource`?

Reply:
> what kind of strict type should I put for `resource`?

PHP... Seems this is - still unresolved - poor typehinting of the language.

Okay, so let's add just PHPDoc:
```php
    /**
     * @param resource $inputStream
     *
     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
     */
```
any opinion @adamwojs @Steveb-p @mikadamczyk ?

Reply:
LGTM.